makefile

```diff
@@ -12,18 +12,30 @@ orgPath := github.com/evergreen-ci
 projectPath := $(orgPath)/$(name)
 # end project configuration
 
+
 # override the go binary path if set
 ifneq (,$(GO_BIN_PATH))
-gobin := $(GO_BIN_PATH)
-else
-gobin := $(shell if [ -x /opt/golang/go1.9/bin/go ]; then echo /opt/golang/go1.9/bin/go; fi)
-ifeq (,$(gobin))
-gobin := go
+  gobin := $(GO_BIN_PATH)
+ else
+  gobin := $(shell if [ -x /opt/golang/go1.13/bin/go ]; then echo /opt/golang/go1.13/bin/go; fi)
```

Comment:
Do we really want to hard code go locations in the makefile. Can we do this without that?